### ISSUE TITLE
Ruckus SmartZone Version MIBs for Connected APs and induvidial AP polling

### ISSUE THREAD

**Issue by tnielsen2** [librenms/librenms#10181]
The recently released Smarzone module is not able to poll connected APs for version 5.0.
Version 3.6 reports correctly for the MIB shows in PR 9727.

I am not aware of how to do this if the version of the SmartZone is polled as 5.0, then it should poll with a different OID.

Smartzone can be polled for its version with the following MIB/OID:
```
TABLE 182 ruckusSZEventFirmwareVersion
Object Name ruckusSZEventFirmwareVersion
Object Ζdentifier .1.3.6.1.4.1.25053.2.11.2.21
Description The SmartZone firmware version.
```

Then the following OID can be polled for connected APs when running version 5
```
Object Name ruckusCtrlSystemNodeNumApConnected
Parent Node ruckusCtrlSystemNodeTable
Object Ζdentifier 1.3.6.1.4.1.25053.1.8.1.1.1.1.1.1.19
Description Number of APs currently connected to this node.
```

Today Cisco WLCs are polled for induvidual APs and have a "is-deleted" database field to alert if a specific AP goes down. This makes alerting easy and simple.

If there was some kind of mapping for the ruckusCtrlApTable MIB for each induvidual AP, then this might be possible as well. 

```
Object Name ruckusCtrlApStatus
Parent Node ruckusCtrlApTable
Object Ζdentifier 1.3.6.1.4.1.25053.1.8.1.1.1.2.1.1.30
Description The AP status type, which is: 0: not available (busy or not running) 1: connected
```

Let me know how else I can help with improving the SmartZone Ruckus module!

**Comment by PipoCanaja** [comment]
Please provide all the requested data and, next time, please keep the "issue template". From that, you can either start coding/implementing it, or wait for somebody to do it. 

**For help and support you can use our [Discord server](https://t.libren.ms/discord) or our [community site](https://community.librenms.org).**



Please confirm each of the sections below by putting an `x` in the box like `[x]`.

- [ ] Please provide **ALL** info asked for [here](http://docs.librenms.org/Support/FAQ/#faq20).
- [ ] Please provide as much detail as possible.
- [ ] Please do NOT post more than 10 lines of debug information here, use a pastebin service or GitHub Gists.

**Comment by djamp42** [comment]
I did the original sz, i only had 3.6 to work with.. What is the snmp sysobjectid for SZ 5.0? I checked and the connected APs is the same exact OID, so that should work. Is it being recognized as ruckus-sz?

**Comment by eimann** [comment]
Hey, maybe PR #10205 helps!

**Comment by tnielsen2** [comment]
@djamp42 

Sorry for the late response. 

My Smartzones are being recognized as ruckus-sz. It can poll SSIDs and connected clients, and "Total APs" with no issue, but the SZ running 5.

If I poll the sysobjectid I get the following:
```
➜  ~ snmpwalk -Os -c public -v 2c 10.8.21.34 1.3.6.1.2.1.1.1
sysDescr.0 = STRING: SmartZone
```

If I poll 1.3.6.1.4.1.25053.1.8.1.1.1.1.8.1.1, I get the total count of APs, to the tune of 35 entires. Problem is, I have 34 connected with 1 down. 

if I poll 1.3.6.1.4.1.25053.1.8.1.1.1.1.5.1.9, I get an integer count of connected APs for each zone. 
If I add the integers for all of these entires, I get the total number I am looking for, which is 34.


The following OID is used by LibreCode for connected APs. I tested this on my 5.0 controller and it doesn't work. 
```
➜  ~ snmpwalk -Os -c public -v 2c 10.8.21.34 .1.3.6.1.4.1.25053.1.8.1.1.1.1.1.1.19.
No log handling enabled - using stderr logging
.1.3.6.1.4.1.25053.1.8.1.1.1.1.1.1.19.: Unknown Object Identifier (Sub-id not found: enterprises -> )
```
The following OID is used by LibreCode for total APs. I tested this on my 5.0 controller and it works. 
```
➜  ~ snmpwalk -Os -c public -v 2c 10.8.21.34 .1.3.6.1.4.1.25053.1.4.1.1.1.15.1.0
enterprises.25053.1.4.1.1.1.15.1.0 = Gauge32: 35
```

If I poll the following OID, I get information that is returned for all APs within the administrative domain. 

➜  ~ snmpwalk -Os -c public -v 2c 10.8.21.34 1.3.6.1.4.1.25053.1.8.1.1.1.1.2
enterprises.25053.1.8.1.1.1.1.2.1.1.139.32.129.213.150.98.64.217.163.219.42.60.244.221.227.247 = Hex-STRING: 8B 20 81 D5 96 62 40 D9 A3 DB 2A 3C F4 DD E3 F7
enterprises.25053.1.8.1.1.1.1.2.1.2.139.32.129.213.150.98.64.217.163.219.42.60.244.221.227.247 = STRING: "Administration Domain"
enterprises.25053.1.8.1.1.1.1.2.1.3.139.32.129.213.150.98.64.217.163.219.42.60.244.221.227.247 = ""
enterprises.25053.1.8.1.1.1.1.2.1.8.139.32.129.213.150.98.64.217.163.219.42.60.244.221.227.247 = INTEGER: 0
enterprises.25053.1.8.1.1.1.1.2.1.9.139.32.129.213.150.98.64.217.163.219.42.60.244.221.227.247 = INTEGER: 7
enterprises.25053.1.8.1.1.1.1.2.1.10.139.32.129.213.150.98.64.217.163.219.42.60.244.221.227.247 = INTEGER: 34
enterprises.25053.1.8.1.1.1.1.2.1.11.139.32.129.213.150.98.64.217.163.219.42.60.244.221.227.247 = INTEGER: 1

The integer of 7 indicates how many zones exist within the cluster. 
The integer of 34 indicates the connected APs to the cluster. 
The integer of 1 indicates the number of disconnected APs from the cluster.

There are other OIDs that are polled that are controller specific, but since APs can move from controller to controller, having a total cluster poll would be beneficial.
